addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/handler/GlobalCacheHandler.java

```diff
@@ -66,14 +71,17 @@
 
     private InetAddress ifAddress;
     private CommandProcessor commandProcessor;
-    ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(MAX_GC_DEVICES);
-    ScheduledFuture<?> scheduledFuture;
+    private ScheduledExecutorService scheduledExecutorService = Executors.newScheduledThreadPool(MAX_GC_THREADS);
```

Comment:
Cool.  Then I think one pool per handler will solve the problem and not be wasteful.